### PR TITLE
DOCS: Upgrade Sphinx Theme to version 1.x

### PR DESCRIPTION
**Pull request by lauranovich**
This PR upgrades the Sphinx theme to version 1.x

### CODE REVIEW

**Comment by lauranovich** [comment]
@dgarcia360 can you please review?

File: .github/workflows/docs-pages@v2.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "Publish docs to GitHub Pages"
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "docs/**"
```
Comment (dgarcia360) [inline .github/workflows/docs-pages@v2.yaml]:
```suggestion
on:
  push:
    branches:
    - scylla-3.x
    paths:
    - 'docs/**'
    - 'faq/**'
    - 'manual/**'
    - 'changelog/**'
    - 'upgrade_guide/**'
```

Comment (dgarcia360) [reply .github/workflows/docs-pages@v2.yaml]: This project does not have a master branch, it's named "scylla-3.x".

File: .github/workflows/docs-pr@v1.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: "Build docs on pull requests"
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "docs/**"
```
Comment (dgarcia360) [inline .github/workflows/docs-pr@v1.yaml]:
```suggestion
on:
  pull_request:
    branches:
    - scylla-3.x
    paths:
    - 'docs/**'
    - 'faq/**'
    - 'manual/**'
    - 'changelog/**'
    - 'upgrade_guide/**'
```

File: .github/workflows/docs-pages@v2.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```
Comment (dgarcia360) [inline .github/workflows/docs-pages@v2.yaml:26]:
```suggestion
      - name: Set up Python
        uses: actions/setup-python@v1
        with:
          python-version: 3.7
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
```

Comment (dgarcia360) [reply .github/workflows/docs-pages@v2.yaml:26]: This repo requires JDK to build their reference documentation.

File: docs/source/conf.py [hunk 4]
```diff
@@ -145,13 +145,8 @@ class DitaaLexer(BashLexer):
...
 html_theme_options = {
     'branch_substring_removed': 'scylla-',
-    'header_links': [
-    ('Scylla Java Driver', 'https://java-driver.docs.scylladb.com/'),
-    ('Scylla Cloud', 'https://docs.scylladb.com/scylla-cloud/'),
-    ('Scylla University', 'https://university.scylladb.com/'),
-    ('ScyllaDB Home', 'https://www.scylladb.com/')],
     'github_issues_repository': 'scylladb/java-driver',
-    'show_sidebar_index': True,
+    'hide_sidebar_index': 'False',
```
Comment (dgarcia360) [inline docs/source/conf.py]:
```suggestion
    'hide_sidebar_index': 'false',
```

Should work as is too, but best if we keep it in lowercase as we have done in other projects.

**Review by dgarcia360 — APPROVED** [review]

**Review by tzach — APPROVED** [review]

**Review by dgarcia360 — COMMENTED** [review]
(comment deleted)

**Review by dgarcia360 — CHANGES_REQUESTED** [review]
This PR was replaced by https://github.com/scylladb/java-driver/pull/103, please close this one.

**Comment by tzach** [comment]
@avelanarius can you please take a look?

**Pull request closed without merging**